vision/cloud-client/src/test/java/com/example/vision/DetectIT.java

```diff
@@ -227,4 +224,78 @@ public void testPropertiesGcs() throws Exception {
     assertThat(got).contains("g:");
     assertThat(got).contains("b:");
   }
+
+  @Test
+  public void detectWebAnnotations() throws Exception {
+    // Act
+    String[] args = {"web", "./resources/landmark.jpg"};
+    Detect.argsHelper(args, out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("Palace");
+    assertThat(got).contains("Rom"); // Matches Rome and Roman
+  }
+
+  @Test
+  public void detectWebAnnotationsGcs() throws Exception {
+    // Act
+    String[] args = {"web", "gs://cloud-samples-tests/vision/landmark.jpg"};
+    Detect.argsHelper(args, out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("Palace");
+    assertThat(got).contains("Rom"); // Matches Rome and Roman
+  }
+
+  @Test
+  public void testCropHints() throws Exception {
+    // Act
+    String[] args = {"crop", "./resources/wakeupcat.jpg"};
+    Detect.argsHelper(args, out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("vertices {");
+    assertThat(got).contains("x: 599");
+    assertThat(got).contains("y: 475");
+  }
+
+  @Test
+  public void testCropHintsGcs() throws Exception {
+    // Act
+    String[] args = {"crop", "gs://cloud-samples-tests/vision/wakeupcat.jpg"};
+    Detect.argsHelper(args, out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("vertices {");
+    assertThat(got).contains("x: 599");
```

Comment:
It's entirely possible that the API result could change over time for these. My expectation however is that the crop hints from this image should match the bounds of the image because it is already framed and cropped.